recipes/tiny-aes-c/all/conanfile.py

```diff
@@ -91,10 +108,5 @@ def package(self):
 
     def package_info(self):
         self.cpp_info.libs = ["tiny-aes"]
-        self.cpp_info.defines.append(str(self.options.aes_block_size))
-        if self.options.cbc:
-            self.cpp_info.defines.append("CBC")
-        if self.options.ecb:
-            self.cpp_info.defines.append("ECB")
-        if self.options.ctr:
-            self.cpp_info.defines.append("CTR")
+        for flag in self._cflags:
+            self.cpp_info.defines.append(flag)
```

Comment:
broken, `self._cflags` is an empty array for consumers who don't build from source

Reply:
it needs to be set to `1` as well https://github.com/conan-io/conan-center-index/pull/4867#discussion_r596314882